src/main/java/org/wso2/lsp4intellij/contributors/rename/LSPRenameHandler.java

```diff
@@ -66,13 +66,13 @@ public void invoke(@NotNull Project project, @NotNull PsiElement[] elements, Dat
     public void invoke(@NotNull Project project, Editor editor, PsiFile file, DataContext dataContext) {
         EditorEventManager manager = EditorEventManagerBase.forEditor(editor);
         if (manager != null) {
-            if (editor.getContentComponent().hasFocus()) {
-                LSPPsiElement psiElement = getElementAtOffset(manager,
-                        editor.getCaretModel().getCurrentCaret().getOffset());
-                if (psiElement != null) {
-                    doRename(psiElement, editor);
-                }
+            // if (editor.getContentComponent().hasFocus()) {
+            LSPPsiElement psiElement = getElementAtOffset(manager,
+                editor.getCaretModel().getCurrentCaret().getOffset());
+            if (psiElement != null) {
+                doRename(psiElement, editor);
             }
+            // }
```

Comment:
Shall we remove the commented lines here? :)

Reply:
Yes, please. I make it commented just to make the point and make it clear to understand. We should definitely remove the commented lines. :)